app/views/layouts/application.html.erb

```diff
@@ -10,12 +10,21 @@
 
   <body>
     <nav class = "topnav">
+      <%= link_to "Home", "/"%>
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
-      <%= link_to "Cart: #{cart.total_items}", "/cart" %>
-      <%= link_to "Home", "/"%>
-      <%= link_to "Log In", "/login"%>
-      <%= link_to "Register", "/register"%>
+      <section class="right-pane">
+        <%= link_to "Cart: #{cart.total_items}", "/cart" %>
+      </section>
```

Comment:
Nice job getting it to align to the right! I added an issue #6 for the rest of the links.